Application/src/visuals/handlers/Animation.java

```diff
@@ -4,36 +4,38 @@
 import visuals.Draw;
 
 /**
- * Handles refreshing and animating draw objects.7
- * @version 0.1
+ * Handles animation requests to the Draw object.
+ * @version 1.0
  * @author Pontus Laestadius
  */
 public class Animation extends Thread {
-    Draw item;
-
-    public Animation(Draw item) {
-        this.item = item;
-    }
+    private static Thread th;
```

Comment:
maybe aniThread?

Reply:
Thread th is a pretty standard implementation you see in most examples of runnables.

Reply:
Examples yes, but does that occur in deployment versions?

Reply:
+1